BGL/test/BGL/test_cgal_bgl_named_params.cpp

```diff
@@ -97,6 +97,7 @@ void test(const NamedParameters& np)
   assert(get_param(np, CGAL::internal_np::face_normal).v == 36);
   assert(get_param(np, CGAL::internal_np::random_seed).v == 37);
   assert(get_param(np, CGAL::internal_np::do_project).v == 38);
+  assert(get_param(np, CGAL::internal_np::opengr_options).v == 9001);
```

Comment:
this one no longer exists

Reply:
I'll update the test with the new NP and remove this one.